tutorials/getting-started/layout-widgets.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Layout (nested) widgets"
+layout: tutorial
+---
+
+## Layout (nested) widgets
+
+You’ve built a couple of custom widgets, you’ve got a powerful `area` running down the main column of your template, awesome. Now you want start pairing widgets up, creating complex layouts of widgets. `singletons` let you enforce particular widgets in particular pieces of layout, but you want the editor to decide where it's appropriate to create these complex arrangements and be able to mix them with simpler widgets.
+
+You want a widget that creates a layout of inner widgets.
+
+### Create a layout widget
+
+Generally, you need a dedicated widget to create layout. The Apostrophe team sometimes refers to these as ‘layout widgets’ that give form and structure to ‘content widgets’ (widgets whose sole job is to display content). **In this example we'll make a simple two column layout.**
+
+With the `apostrophe-cli` installed
```

Comment:
This should be a link. And I'd suggest wording like "We'll use apostrophe-cli to make a new widget quickly."